internal/configs/configschema/internal_validate_test.go

```diff
@@ -134,6 +134,32 @@ func TestBlockInternalValidate(t *testing.T) {
 			},
 			[]string{"foo: either Type or NestedType must be defined"},
 		},
+		"attribute with both type and nestedtype": {
```

Comment:
tiny tiny tiny nitpick: Could we make the test name a little more verbose to describe what it's doing here?

Maybe something like `attribute with both type and nestedtype should not suppress other validation messages`

I know it doesn't really fit with the naming of the rest of these, but I feel like it would help us understand why we added this test in the future if we ever regress.

I would also be open to a comment that describes what you want to do for this individual test.